rust-sdk/core/src/types/tick.rs

```diff
@@ -0,0 +1,35 @@
+#![allow(non_snake_case)]
+
+#[cfg(feature = "wasm")]
+use serde_big_array::BigArray;
+
+#[cfg(feature = "wasm")]
+use orca_whirlpools_macros::wasm_expose;
+
+use crate::TICK_ARRAY_SIZE;
+
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Default)]
+#[cfg_attr(feature = "wasm", wasm_expose)]
+pub struct TickRange {
+    pub tick_lower_index: i32,
+    pub tick_upper_index: i32,
+}
+
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Default)]
+#[cfg_attr(feature = "wasm", wasm_expose)]
+pub struct TickFacade {
+    pub initialized: bool,
+    pub liquidity_net: i128,
+    pub fee_growth_outside_a: u128,
```

Comment:
Yeah removed it cause it isn't currently needed. Thinking about it is better to add the fields even if we don't need it now (since introducing later would be a breaking change indeed)